fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchURITest.java

```diff
@@ -49,14 +56,25 @@ public void testSearchURI_uri_chained() throws Exception {
         assertSearchReturnsComposition("subject:Basic.uri", "http://hl7.org/fhir/DSTU2");
         assertSearchReturnsComposition("subject:Basic.uri", "urn:uuid:53fefa32-1111-2222-3333-55ee120877b7");
         
+        // https://github.com/IBM/FHIR/issues/273
         // Matches are supposed to be precise (e.g. case, accent, and escape sensitive), but aren't
-        // 
+        
 //        assertSearchDoesntReturnComposition("subject:Basic.uri", "http://HL7.org/FHIR/dstu2");
 //        assertSearchDoesntReturnComposition("subject:Basic.uri", "urn:uuid:53FEFA32-1111-2222-3333-55EE120877B7");
```

Comment:
why not just remove them?

Reply:
because we want exactly these lines after we implement #273 ...this is also why i left the link to that issue just above the comment